src/commands/QueryCommand.ts

```diff
@@ -56,7 +56,9 @@ export class QueryCommand implements yargs.CommandModule {
             console.log(chalk.green("Running query: ") + PlatformTools.highlightSql(args._[1]));
             const queryResult = await queryRunner.query(args._[1]);
             console.log(chalk.green("Query has been executed. Result: "));
-            console.log(PlatformTools.highlightJson(JSON.stringify(queryResult, undefined, 2)));
+            console.log(PlatformTools.highlightJson(
+                queryResult === undefined ? "undefined" : JSON.stringify(queryResult, undefined, 2)
```

Comment:
such logic isn't obvious at all. Maybe instead of showing undefined, we just tell something like "Query has been executed. No result was returned" instead?

Reply:
Sure - the big issue here is that passing `undefined` to `JSON.stringify` currently returns back an `undefined` value causing a crash - `highlightJson` shouldn't accept undefined.